packages/flutter/lib/src/foundation/tree_diagnostics_mixin.dart

```diff
@@ -2,9 +2,1438 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
 import 'package:meta/meta.dart';
 
-import 'print.dart';
+/// Styles for displaying a node in a [DiagnosticsNode] tree.
+///
+/// See also:
+///
+///  * [DiagnosticsNode.toStringDeep], which dumps text art trees for these
+///    styles.
+///  * [_normalTextConfiguration], which shows what text art for the standard
```

Comment:
The docs shouldn't really mention privates at all. It doesn't help the reader: they can't do anything with them. It's just taunting the developer. :-)